tests/errorhandling/closing_files.go

```diff
@@ -0,0 +1,23 @@
+package errorhandling
+
+import (
+	"os"
+
+	"golang.org/x/sys/unix"
+)
+
+func SafelyCloseFile(f *os.File) {
```

Comment:
I don't think the name matters too much at this point and I wuld leave it as it, but other naming considerations would be putting more in the foreground that this says that we assume a read only file being closed. I have not smart ideas though.